mobile/src/main/java/ch/epfl/sdp/mobile/application/StoreDocuments.kt

```diff
@@ -7,11 +7,15 @@ package ch.epfl.sdp.mobile.application
  * @param name the human-readable name associated to this profile.
  * @param emoji the emoji associated with this profile.
  * @param backgroundColor the hex color code for this profile.
+ * @param uid the unique identifier for this profile.
+ * @param following a list of unique identifiers of the profiles this profile is following.
  */
 data class ProfileDocument(
     val name: String? = null,
     val emoji: String? = null,
-    val backgroundColor: String? = null
+    val backgroundColor: String? = null,
+    val uid: String? = null,
```

Comment:
The `uid` field should be annotated with `@DocumentId` (see the documentation [here](https://firebase.google.com/docs/reference/android/com/google/firebase/firestore/DocumentId). Additionally, the `FakeStore` implementation should be updated to handle `@DocumentId`-annotated fields.